R/summaryStats.R

```diff
@@ -1,221 +1,205 @@
-
-#' Biomarker Data
+#' Biomarker Stats
 #'
-#' Does some stuff
+#' Computes summary statistics National Micronutrient Surveys
 #'
 #' @param theData the data
 #' @param groupId the group
 #' @param biomarkerField the field name of the biomarker measurement data
 #' @param aggregationField the field name to aggregate by
 #' @param thresholds the thresholds
-#'
+#' 
 #' @importFrom magrittr %>%
-#'
+#' @import srvyr jtools survey dplyr
+NULL
```

Comment:
Is this `NULL` intended / necessary?

Reply:
I think it's not needed. Perhaps later when we use @docType for the package documentation. But for now it's not. Thanks for deleting!